src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Figure[] figures = new Figure[] {
+            new Circle(new Random().nextInt(100), ColorSupplier.randomColor()),
+            new Triangle(new Random().nextInt(100), ColorSupplier.randomColor()),
+            new Square(new Random().nextInt(100), ColorSupplier.randomColor()),
+            new Rectangle(new Random().nextInt(100), new Random().nextInt(100),
+                    ColorSupplier.randomColor()),
+            new IsoscelesTrapezoid(new Random().nextInt(100), new Random().nextInt(100),
+                    new Random().nextInt(100), ColorSupplier.randomColor())};
+
+    public Figure getRandomFigure() {
+        return figures[new Random().nextInt(figures.length)];
+    }
```

Comment:
What result will I get if I call the `getRandomFigure()` method twice?

Reply:
You will get two figures with random parameters.